app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> None:
+    file_names = command.replace("cp ", "").split(" ")
+    if file_names[0] == file_names[1] or not command.startswith("cp"):
```

Comment:
Please  first make sure you check that the input string contains 3 elements separated by space, and has `cp` on a first place.